sdk/test/NetStandard/UnitTests/Core/AWSSDKUtilsTests.cs

```diff
@@ -0,0 +1,29 @@
+using Amazon.Util;
+using Xunit;
+
+namespace UnitTests.NetStandard.Core
+{
+    [Trait("Category", "Core")]
+    public class AWSSDKUtilsTests
+    {
+        [Theory]
+        [InlineData("value, with special chars!", "value%2C%20with%20special%20chars%21")]
```

Comment:
Can we add some tests with multi byte characters?

Reply:
Done